backend/tests/endpoints/test_room.py

```diff
@@ -82,6 +82,31 @@
     },
 }
 
+leave_room_url = (
+    "api/v1/org/3467sd4671a5f5478df56u911/rooms/23dg67l0eba8adb50ca13a24/"
+    + "members/619baa5c1a5f54782939d386"
+)
+
+remove_room_member_url = (
+    "api/v1/org/3467sd4671a5f5478df56u911/rooms/23dg67l0eba8adb50ca13a24/"
+    + "members/61696f5ac4133ddaa309dcfe?admin_id=6169704bc4133ddaa309dd07"
+)
+
+fake_member_leave_room_url = (
+    "api/v1/org/3467sd4671a5f5478df56u911/rooms/23dg67l0eba8adb50ca13a24/"
+    + "members/1696f5ac4133ddaa309dcfe?admin_id=6169704bc4133ddaa309dd07"
+)
+
+fake_member_remove_room_member_url = (
+    "api/v1/org/3467sd4671a5f5478df56u911/rooms/23dg67l0eba8adb50ca13a24/"
+    + "members/1696f5ac4133ddaa309dcfe"
+)
+
+non_admin_remove_room_member_url = (
+    "api/v1/org/3467sd4671a5f5478df56u911/rooms/23dg67l0eba8adb50ca13a24/"
+    + "members/61696f5ac4133ddaa309dcfe?admin_id=619baa5c1a5f54782939d386"
+)
```

Comment:
too many urls for one endpoint @kinglighthill, if we go on creating a url for each test case we will end up having more urls than actual test codes.
Since the urls differ only slightly in characters, I suggest we use one url for each endpoint and use string methods to modify the characters within each test case when needed. This way we can reduce unnecessary repetitions. Thanks